readme.md

```diff
@@ -57,12 +60,21 @@ Model-driven APIs for simplified programmability of your network device. Generat
 - [Napalm](https://github.com/napalm-automation/napalm)  
 Network Automation and Programmability Abstraction Layer with Multivendor support
 
+- [Paramiko](http://www.paramiko.org/)  
+Implementation of SSHv2 protocol, providing both client and server functionality.
+
 - [Netmiko](https://github.com/ktbyers/netmiko)  
-Multi-vendor library to simplify Paramiko SSH connections to network devices
+Multi-vendor library to simplify Paramiko SSH connections to network devices.
 
 - [Trigger](https://github.com/trigger/trigger)  
 "... a robust network automation toolkit written in Python that was designed for interfacing with network devices."
 
+- [CiscoConfigParse](http://www.pennington.net/py/ciscoconfparse/)  
```

Comment:
We should change this link the repository => https://github.com/mpenning/ciscoconfparse